inc/class-shortcode-ui.php

```diff
@@ -244,9 +268,13 @@ public function enqueue() {
 				'insert_content_label'              => __( 'Insert Content', 'shortcode-ui' ),
 			),
 			'nonces'     => array(
-				'preview'        => wp_create_nonce( 'shortcode-ui-preview' ),
+				'wp_rest'        => wp_create_nonce( 'wp_rest' ),
```

Comment:
Unless I'm missing something, we're not checking nonces on any of the REST request handlers. Should we be?

I think there might be a benefit to making this unauthenticated, as then it would be easier to reuse the functionality in external editors and the like. However, that requires a much more solid approach to security, as it opens an attack surface for anyone to compel a server to render an arbitrary shortcode. Given that people still use things like `[php]` shortcodes, this gets sketchy pretty fast.